plugins/BEdita/Core/src/Shell/StreamsShell.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2019 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+namespace BEdita\Core\Shell;
+
+use BEdita\Core\Model\Entity\Stream;
+use Cake\Console\Shell;
+use Cake\I18n\Date;
+use Cake\ORM\TableRegistry;
+
+/**
+ * Stream shell commands: removeOrphans
+ *
+ * @since 4.0.0
+ */
+class StreamsShell extends Shell
+{
+
+    /**
+     * {@inheritDoc}
+     *
+     * @codeCoverageIgnore
+     */
+    public function getOptionParser()
+    {
+        $parser = parent::getOptionParser();
+        $parser->addSubcommand('removeOrphans', [
+            'help' => 'remove obsolete/orphans streams and related files',
+            'parser' => [
+                'description' => [
+                    'Remove orphans streams.'
+                ],
+                'options' => [
+                    'days' => [
+                        'help' => 'Days to consider for stream research for orphans',
+                        'required' => false,
+                        'default' => 1,
+                    ],
+                ],
+            ]
+        ]);
+
+        return $parser;
+    }
+
+    /**
+     * Create a new Stream
+     *
+     * @return void
+     */
+    public function removeOrphans()
+    {
+        $days = (int)$this->param('days');
+        $streamsTable = TableRegistry::get('Streams');
```

Comment:
Use `$modelClass` instead, and you'll have `$this->Streams` available. See https://github.com/bedita/bedita/blob/4-cactus/plugins/BEdita/Core/src/Shell/JobsShell.php#L31

This is better because on newer versions of CakePHP, `TableRegistry::get()` has been deprecated.